trivy-image-scan/action.yml

```diff
@@ -53,6 +53,13 @@ runs:
       run: |
         echo "FORMAT=sarif" >> $GITHUB_ENV
         echo "OUTPUT=''trivy-results.sarif''" >> $GITHUB_ENV
+    
+    - name: Set up output values
+      if: ${{ inputs.output-mode == 'json' }}
```

Comment:
Could you update the description on line 31 to include the new output mode

Reply:
updated